### PR TITLE
limit review queues to upcoming due_dates

### LINKED ISSUE #20622: Reduce the visible depth of the manual review Queue
### Describe the problem and steps to reproduce it:
The current manual review queue displays all extensions that have a due date including those that have a due date in several days.

### What did you expect to happen?
In order to reduce the mental overload of the reviewers we would like to restrict the number of extensions visible to the reviewer and display only those that are due for the review within next two business days. The total number of extensions should be displayed properly.

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #20622

### CODE REVIEW

**Comment by wagnerand** [comment]
@eviljeff Assuming the following scenario:

A reviewer has permission `Addons:TriageDelayed` but not `Addons:AllDueDates`. Config `upcoming-due-date-cut-off-days` is set to 1.

Will that reviewer see unlisted add-ons to be triaged with a due date further out than 1 day?

File: src/olympia/addons/models.py [hunk 1]
```diff
@@ -91,6 +92,7 @@
 MAX_SLUG_INCREMENT = 999
 SLUG_INCREMENT_SUFFIXES = set(range(1, MAX_SLUG_INCREMENT + 1))
 GUID_REUSE_FORMAT = 'guid-reused-by-pk-{}'
+UPCOMING_DUE_DATE_CUT_OFF_DAYS_CONFIG_KEY = 'upcoming-due-date-cut-off-days'
```
Comment (diox) [inline src/olympia/addons/models.py:95]: Do we have to have a config ? It adds a SQL query...

Comment (eviljeff) [reply src/olympia/addons/models.py:95]: That was the request from @wagnerand as far I understood it.

**Comment by eviljeff** [comment]
> @eviljeff Assuming the following scenario:
> 
> A reviewer has permission `Addons:TriageDelayed` but not `Addons:AllDueDates`. Config `upcoming-due-date-cut-off-days` is set to 1.
> 
> Will that reviewer see unlisted add-ons to be triaged with a due date further out than 1 day?

It's not explicitly tested, but if the `due_date`s of the delayed add-ons (or any version in the queue) is further out than the cut off then no.

If this isn't what was expected can you explain what should happen? (The more different permissions, the more this is a possibility...)

File: src/olympia/addons/models.py (hunk 1 above)
Comment (wagnerand) [reply src/olympia/addons/models.py:95]: Yes please. 2 days is too much at the moment, and this allows us to be flexible without a code change and deploy.

Comment (diox) [reply src/olympia/addons/models.py:95]: We used to have caching for the `get_config` stuff but it was often inconsistent and not that useful so we removed it. I don't think it's particularly worth doing for single-key lookups like that. It's fine we can live with the extra (short) query.

**Comment by kplotnik** [comment]
@wagnerand @eviljeff Ideally we want to show the `Addons:TriageDelayed` extensions independent from the `upcoming-due-date-cut-off-days` but I think for the first iteration we can use the current solution. And then we will see how it evolves.

**Pull request merged**